src/io/utils.rs

```diff
@@ -48,21 +47,17 @@ impl<S: PolyHashSampler<[u8; 32]>> PublicSampledData<S> {
         let d = obf_params.d;
         let level_size = (1u64 << obf_params.level_width) as usize;
         let mut rs = Vec::with_capacity(level_size);
-        let mut rgs = Vec::with_capacity(level_size);
         let one = S::M::identity(params, 1, None);
-        let gadget_d_plus_1 = S::M::gadget_matrix(params, d + 1);
         for i in 0..level_size {
             let tag = format!("R_{}", i).into_bytes();
             let r_i_bar = hash_sampler.sample_hash(params, hash_key, &tag, d, d, DistType::BitDist);
             let r_i = r_i_bar.concat_diag(&[&one]);
-            let rg = r_i.clone() * &gadget_d_plus_1;
             rs.push(r_i);
-            rgs.push(rg);
         }
 
         let log_base_q = params.modulus_digits();
         let dim = params.ring_dimension() as usize;
-        // input bits, poly of the RLWE key
+        // input bits, poly of the RLWE key, it contains constant 1
```

Comment:
I think we can say that `packed_input_size` refers to:
- number of polynomials used to pack the evaluator's input bits 
- 1 for the RLWE key (t)